.github/workflows/rerunner.yml

```diff
@@ -93,22 +93,24 @@ jobs:
           AZURE_STORAGE_ACCOUNT: ${{ secrets.AZURE_STORAGE_ACCOUNT }}
           AZURE_STORAGE_CONTAINER: ${{ inputs.storage_container }}
 
-      - name: Setup test environment
-        run: |
-          python ./scripts/vendors/microsoft/setup_environment.py --source ${{ github.workspace }}/artifacts --destination ${{ github.workspace }}/jdk
-
       - name: Configure linux
         if: ${{ runner.os == 'linux' }}
         uses: ./.github/vendor-templates/microsoft/configure_linux
 
+      - name: Setup test environment
+        run: |
+          python ./scripts/vendors/microsoft/setup_environment.py --source ${{ github.workspace }}/artifacts --destination ${{ github.workspace }}/jdk
+
       - name: Run ${{ inputs.suite }} ${{ inputs.aqa-targets }}
         uses: adoptium/run-aqa@v2.0.1
+        env:
+          TESTLIST: ${{ inputs.aqa-targets }}
         with:
           jdksource: 'customized'
           aqa-testsRepo: ${{ inputs.aqatest-repo }}
           build_list: ${{ inputs.suite }}
-          target : _testList TESTLIST=${{ inputs.aqa-targets }}
-          version: ${{ inputs.version }}
+          target : _${{ inputs.aqa-targets }}
+          version: ${{ inputs.version }}u
```

Comment:
NIT. Is it possible to send the entire version and the 'u' to in `${{ inputs.version }}`? I think it will make it cleaner.

Reply:
Yeah that makes sense to me

Reply:
done!